libs/design/src/molecules/hero/README.md

```diff
@@ -1,12 +1,16 @@
 # Hero
-`DaffHeroComponent` is a top level container that is large and captivating. It should only be used once as the first container on any given page. It supports transclusion of any content and includes predefined `tagline`, `title` and `subtitle` styles.
+Hero is a top level container that is large and captivating. It should only be used once as the first container on any given page.
 
 ## Supported Content Types
-A `daff-hero` can transclude:
-- `[daffHeroTagline]` - Hero tagline
-- `[daffHeroTitle]` - Hero title, used with the `<h1>` tag
-- `[daffHeroSubtitle]` - Hero subtitle, used with the `<h2>` tag
-- Any additional components
+A `daff-hero` supports transclusion of any content and includes stylized `icon`, `tagline`, `title` and `subtitle` content containers.
+
+| Selector             | Description                                                                            |
+| -------------------- | -------------------------------------------------------------------------------------- |
+| `[daffHeroIcon]`     | Hero icon, intended for visual or branding reinforcement                               |
+| `[daffHeroTagline]`  | Hero tagline                                                                           |
+| `[daffHeroTitle]`    | Hero title, used with the `<h1>` tag                                                   |
+| `[daffHeroSubtitle]` | Hero subtitle, used with the `<h2>` tag                                                |
+| `[daffHeroBody]`     | Container for any additional content, it will not respect the `textAlignment` property |
```

Comment:
But the previous 4 selectors do respect the `textAlignment`? `any additional components or elements` is a little ambiguous. This should be clarified.

In the text align example you set `text-align: center` on the icon, does this interfere with the text align passed as an input to the hero component?